shaka/src/memory/object_tracker.cc

```diff
@@ -163,7 +167,7 @@ void ObjectTracker::DestroyObjects(
   // Don't remove elements from |objects_| until after the destructor so the
   // destructor can call AddRef.
   for (auto it = objects_.begin(); it != objects_.end();) {
-    if (to_delete_.count(it->first) > 0) {
+    if (to_delete_.find(it->first) != to_delete_.end()) {
```

Comment:
I think these are the same.  Even though it's called `count`, there is only ever one element.  But either is fine.

Reply:
`find` stops on first find unlike `count`, and these containers can be rather large